packages/@okta/vuepress-theme-prose/components/OnThisPageItem.vue

```diff
@@ -56,14 +63,7 @@ export default {
       if(hash) {
         const node = document.querySelector(hash);
         if(node) { // node is sometimes null - perhaps content hasn't loaded?
```

Comment:
Any way we can encapsulate some of this checking to the `historyPushAndScrollToAnchor` method? We are also doing the queryselector call inside there. I am also wondering if the `historyPushAndScrollToAnchor` can take a node instead of the string.

Reply:
it can take node, but then `document.querySelector(hash` will be added outside of `historyPushAndScrollToAnchor `, also `historyPushAndScrollToAnchor` reuquire hash string inself to make happen condition comaring `decodeURIComponent(this.$route.hash) !== decodeURIComponent(hash)`